docs/src/user-guide/quick-start/clp-json/main-page.md

```
@@ -0,0 +1,148 @@
+If you'd like to search your query from the web UI, CLP includes a web interface available at [http://localhost:4000](http://localhost:4000) by default (if you changed `webui.host` or `webui.port` in `etc/clp-config.yml`, use the new values).
+
+:::{image} clp-search-ui.png
+:::
+
+The image above shows the search page after running a query. The numbered circles correspond to the following features:
+
+1. The search box is where you can enter your query.
+2. The timeline shows the number of results across the time range of your query.
+   * You can click and drag to zoom into a time range or use the time range filter in (4).
+3. The table displays the search results for your query.
+4. Clicking the <i class="fa fa-bars"></i> icon reveals additional filters for your query.
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: i

(MD033, no-inline-html)